Does not compile on OSX:

ninja -C build 
ninja: Entering directory `build'
[6/10] Compiling c object enet.dir/callbacks.c.o
FAILED: cc '-DHAS_POLL' '-DHAS_FCNTL' '-DHAS_INET_PTON' '-DHAS_INET_NTOP' '-DHAS_MSGHDR_FLAGS' '-DHAS_SOCKLEN_T' '-Wall' '-Wpedantic' '-Winvalid-pch' '-g' '-Ienet.dir' '-I..' '-I.' '-MMD' '-MQ' enet.dir/callbacks.c.o '-MF' 'enet.dir/callbacks.c.o.d' -o enet.dir/callbacks.c.o -c ../callbacks.c
../callbacks.c:6:10: fatal error: 'enet/enet.h' file not found
#include "enet/enet.h"
         ^
1 error generated.

--- a/meson.build
+++ b/meson.build
@@ -1,0 +1,55 @@
+project('enet', 'c')
+cc = meson.get_compiler('c')
+c_args = []
+
+if cc.has_function('gethostbyaddr_r', prefix : '#include <netdb.h>')
+  c_args = c_args + ['-DHAS_GETHOSTBYADDR_R']
+endif
+if cc.has_function('gethostbyname_r', prefix : '#include <netdb.h>')
+  c_args = c_args + ['-DHAS_GETHOSTBYNAME_R']
+endif
+if cc.has_function('poll', prefix : '#include <sys/poll.h>')
+  c_args = c_args + ['-DHAS_POLL']
+endif
+if cc.has_function('fcntl', prefix : '#include <fcntl.h>')
+  c_args = c_args + ['-DHAS_FCNTL']
+endif
+if cc.has_function('inet_pton', prefix : '#include <arpa/inet.h>')
+  c_args = c_args + ['-DHAS_INET_PTON']
+endif
+if cc.has_function('inet_ntop', prefix : '#include <arpa/inet.h>')
+  c_args = c_args + ['-DHAS_INET_NTOP']
+endif
+if cc.has_member('struct msghdr', 'msg_flags', prefix : '#include <sys/socket.h>')
+  c_args = c_args + ['-DHAS_MSGHDR_FLAGS']
+endif
+if cc.has_type('socklen_t', prefix : '#include <sys/types.h>\n#include <sys/socket.h>')
+  c_args = c_args + ['-DHAS_SOCKLEN_T']
+endif
+
+src = [
+  'callbacks.c',
+  'compress.c',
+  'host.c',
+  'list.c',
+  'packet.c',
+  'peer.c',
+  'protocol.c',
+  'unix.c',
+  'win32.c',
+]
+
+incdir = include_directories('include')
+if get_option('shared_lib')
+  enetlib = shared_library(
+    'enet',
+    sources : src,
+    include_directories : incdir,
+    c_args : c_args)
+else
+  enetlib = static_library(
+    'enet',
+    sources : src,
+    include_directories : incdir,
+    c_args : c_args)
+endif

--- a/meson_options.txt
+++ b/meson_options.txt
@@ -1,0 +1,1 @@
+option('shared_lib', type : 'boolean', value : false)

--- a/upstream.wrap
+++ b/upstream.wrap
@@ -1,0 +1,6 @@
+[wrap-file]
+directory = enet-1.3.13
+
+source_url = http://enet.bespin.org/download/enet-1.3.13.tar.gz
+source_filename = enet-1.3.13.tar.gz
+source_hash = e36072021faa28731b08c15b1c3b5b91b911baf5f6abcc7fe4a6d425abada35c

